Makefile

```diff
@@ -84,6 +84,21 @@ build-guacamole-image:
 build-mlflow-image:
 	$(call build_image,"mlflow-server","templates/workspace_services/mlflow/mlflow-server/version.txt","templates/workspace_services/mlflow/mlflow-server/docker/Dockerfile","templates/workspace_services/mlflow/mlflow-server")
 
+firewall-install:
+	make bundle-build DIR=./templates/shared_services/firewall/ \
```

Comment:
please convert all internal make calls to $(MAKE)

Reply:
Sorry, I don't understand what do you mean by that. Do you mean `$(call)`? If so, then I don't think I can set the values of env, such as `DIR` here, in the same statement